### DISCUSSION TITLE
I get No Internet when I turn on Offline Chrome

### DISCUSSION THREAD

**Discussion by thangdangblog** [quasarframework/quasar#10188] [CLI - PWA mode]
I added PWA Mode to create Service Work for my Project. But when I  turn on "Offline" by Chrome Option, I got "No Internet".  
Some errors I get:
- Precaching dit did not find a match for /
- Uncaught (in promise) non-precached-url: crateHandlerBoundToURL('.index.html') was called, but that URL is not precached. Please pass in a URL that is precached instead.

I installed SSL for Local,...

**Comment by ontwikkelfabriek** [comment]
@thangdangblog 

> But when I turn on "Offline" by Chrome Option, I got "No Internet".
Some errors I get:

What if you deploy the PWA Quasar app on a 'real' domain?

> I installed SSL for Local...
What's your local(ssl) full url for the quasar app?

I would first run it on a domain and see if that solved the problem. If not then I would run a Lighthouse diagnostic ( chrome dev tools).

**Reply by hawkeye64** [reply]
I believe it also needs https protocol for it to cache properly. It doesn't work with http.
